### PR TITLE
[4.0] batch language change

### PR DESCRIPTION
**Pull request by brianteeman**
Pull Request for Issue #34874 .

### Summary of Changes
Only shows the batch language change option when the site is a multilingual site

### CODE REVIEW

File: administrator/components/com_categories/tmpl/categories/default_batch_body.php [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 use Joomla\CMS\HTML\HTMLHelper;
 use Joomla\CMS\Language\Text;
+use Joomla\CMS\Language\Multilanguage;
```
Comment (Quy) [inline administrator/components/com_categories/tmpl/categories/default_batch_body.php]:
```suggestion
use Joomla\CMS\Language\Multilanguage;
use Joomla\CMS\Language\Text;
```

**Comment by ChristineWk** [comment]
Result:

screen shot 2021-07-24 at 16 19 39<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34887">issues.joomla.org/tracker/joomla-cms/34887</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 5b220d2ba0dec595d5b35d5172b041ca835958ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34887">issues.joomla.org/tracker/joomla-cms/34887</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 5b220d2ba0dec595d5b35d5172b041ca835958ca<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34887">issues.joomla.org/tracker/joomla-cms/34887</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/34887">issues.joomla.org/tracker/joomla-cms/34887</a>.</sub>

**Comment by chmst** [comment]
Thanks!

**Comment by brianteeman** [comment]
my pleasure

**Pull request merged**